messaging/writer_test.go

```diff
@@ -54,13 +67,26 @@ func TestMissyWriter_WriteError(t *testing.T) {
 
 	writer := missyWriter{brokerWriter: brokerWriterMock}
 
+	exec := false
+
+	monkey.PatchInstanceMethod(reflect.TypeOf(brokerWriterMock), "WriteMessages", func(_ *MockBrokerWriter, ctx context.Context, messages ...Message) error {
+		exec = true
```

Comment:
Eeee why? this is redundant, in above brokerWriterMock.EXPECT().WriteMessages(gomock.Any(), *msg).Return(errors.New("error")) you can pass gomock.Any actually, here you are testing error, it's not important what is inside.